tests/improver-nowcast-extrapolate/04-basic_no_orographic_enhancement.bats

```diff
@@ -33,7 +33,7 @@
 
 @test "extrapolate basic no orographic enhancement" {
   improver_check_skip_acceptance
-  KGO="nowcast-extrapolate/extrapolate_no_orographic_enhancement/kgo.nc"
+  KGO="nowcast-feature-branch/nowcast-extrapolate/kgo_no_orographic_enhancment.nc"
```

Comment:
I don't think this test is relevant for the pysteps method.

Reply:
It's gone.

Reply:
Removed.